thunder/tests/test_interpreter.py

```diff
@@ -1691,6 +1691,40 @@ def fn():
     jit(fn)()
 
 
+def test_zip_lookaside(jit):
+    jitting = False
+
+    def foo(*a, strict=False):
+        return list(zip(*a, strict=strict))
+
+    jfoo = jit(foo)
+    jitting = False
+
+    res1 = foo([1, 2, 3], [4, 5, 6])
+    res2 = foo([1, 2, 3], [4, 5, 6], [7, 8, 9])
+    res3 = foo([1, 2], [4, 5, 6])
+    res4 = foo("abc", "xyz")
+    # , match="zip() argument 2 is longer than argument 1"
+    with pytest.raises(ValueError):
+        res5 = foo([1, 2], [4, 5, 6], strict=True)
```

Comment:
Good to have a test for the error here!

Reply:
We have added the exact text for the error as the match parameter. It worked when we added ```re.escape```